pkg/api/endpoints/util.go

```diff
@@ -28,6 +28,14 @@ import (
 	hashutil "k8s.io/kubernetes/pkg/util/hash"
 )
 
+const (
+	PodHostNamesAnnotation = "endpoints.net.beta.kubernetes.io/hostnames-map"
+)
+
+type HostFQDN struct {
```

Comment:
I don't recall why we called this FQDN if it only holds a hostname, not an FQDN?  Maybe we need a better name?  And a comment on this field.